fourmolu.yaml

```diff
@@ -19,3 +19,9 @@ fixities:
   - infixl 5 ^^^
   # Distribution.Types.InstalledPackageInfo.FieldGrammar
   - infixl 4 <@>
+
+reexports:
```

Comment:
@ffaf1 I went ahead and added the config here, to facilitate https://github.com/fourmolu/fourmolu/pull/346. The styling is still fourmolu 0.12, but fourmolu 0.12 will ignore this config